sidecar/main.cpp

```diff
@@ -156,6 +155,36 @@ int AutoCalibrate(const nlohmann::json& json) {
   return 1;
 }
 
+struct VRFFBData_t {
+  VRFFBData_t(short thumbCurl, short indexCurl, short middleCurl, short ringCurl, short pinkyCurl)
+      : thumbCurl(thumbCurl), indexCurl(indexCurl), middleCurl(middleCurl), ringCurl(ringCurl), pinkyCurl(pinkyCurl){};
+
+  short thumbCurl;
+  short indexCurl;
+  short middleCurl;
+  short ringCurl;
+  short pinkyCurl;
+};
+
+int ServoTest(const nlohmann::json& json) {
+  const bool rightHand = json["right_hand"].get<bool>();
+  const bool extend = json["extend"].get<bool>();
+
+  const short setCurl = extend ? 1000 : 0;
+
+  VRFFBData_t data(setCurl, setCurl, setCurl, setCurl, setCurl);
+
+  const bool success = ConnectAndSendPipe("\\\\.\\pipe\\vrapplication\\ffb\\curl\\" + std::string(rightHand ? "right" : "left"), data);
```

Comment:
Can we make this `ConnectAndSendPipe` method take `T& data` instead of just `T data`? (I couldn't comment above but it bugs me)

Reply:
👍